pkg/plugins/backendplugin/grpcplugin/grpc_plugin.go

```diff
@@ -141,23 +142,23 @@ func (p *grpcPlugin) getPluginClient() (pluginClient, bool) {
 func (p *grpcPlugin) CollectMetrics(ctx context.Context, req *backend.CollectMetricsRequest) (*backend.CollectMetricsResult, error) {
 	pluginClient, ok := p.getPluginClient()
 	if !ok {
-		return nil, backendplugin.ErrPluginUnavailable
+		return nil, plugins.ErrPluginUnavailable
```

Comment:
Please use `.Errorf` here and below to create "actual errors" that includes an error message